setup.cfg

```diff
@@ -34,6 +34,7 @@ install_requires =
     futures; python_version < '3'
     jsonpatch
     cotyledon>=1.5.0
+    redis >= 3.2.0 # MIT
```

Comment:
Redis is not mandatory to run gnocchi, do not make that change.

Reply:
I agree. Moving the change to the incoming redis driver should not make it a requirement here. I've moved it back to the redis section.